cli/internal/cmd/upgradeexecute.go

```diff
@@ -27,13 +29,23 @@ func newUpgradeExecuteCmd() *cobra.Command {
 		RunE:  runUpgradeExecute,
 	}
 
+	// TODO: remove before merging to main. For testing only.
+	cmd.Flags().Bool("helm", false, "DEV ONLY: execute helm upgrade")
+	cmd.Flags().Duration("timeout", 3*time.Minute, "DEV ONLY: change upgrade timeout. Might be useful for slow connections or big clusters.")
```

Comment:
I think we want to keep the flags for now? 
Not sure if the rest of the upgrade operations will be done before v2.4, so we should merge with the flags available.
Maybe remove the `DEV ONLY` and expand on the flag help text.
We could also mark the flags as hidden until we decide on a more permanent solution